pawn/Resources/Callbacks/OnPlayer/OnPlayerCommandText.pwn

```diff
@@ -906,6 +918,42 @@ public OnPlayerCommandText(playerid, cmdtext[]) {
         return 1;
     }
 
+    if (strcmp(cmd, "/vips", true) == 0) {
```

Comment:
Wouldn't it be nice to have this in JavaScript instead?

We'd like to move all code to JavaScript

Reply:
Um maybe for the moment? can we keep pwn. I'll make js for both /admins and /vip altogether.